# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavcodec/moz.build
# Commit: 5147903580b8
# Full Hash: 5147903580b8f39197d67bddd5a0fb91d3670da4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-27 04:32:10
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r 0aad3393b572 -r 5147903580b8 media/ffvpx/libavcodec/moz.build
--- a/media/ffvpx/libavcodec/moz.build	Mon Feb 26 16:07:07 2024 +0000
+++ b/media/ffvpx/libavcodec/moz.build	Mon Feb 26 16:07:08 2024 +0000
@@ -22,6 +22,7 @@
     'allcodecs.c',
     'avcodec.c',
     'avdct.c',
+    'avfft.c',
     'avpacket.c',
     'bitstream.c',
     'bitstream_filters.c',
@@ -145,11 +146,6 @@
   CXXFLAGS += CONFIG["MOZ_LIBVPX_CFLAGS"]
   OS_LIBS += CONFIG["MOZ_LIBVPX_LIBS"]
 
-if CONFIG['MOZ_LIBAV_FFT']:
-    SOURCES += [
-        'avfft.c',
-    ]
-
 SYMBOLS_FILE = 'avcodec.symbols'
 NoVisibilityFlags()
 